src/legacy/core_plugins/kibana/ui_setting_defaults.js

```diff
@@ -269,6 +269,16 @@ export function getUiSettingDefaults() {
       }),
       category: ['discover'],
     },
+    'doc_table:hideTimeColumn': {
+      name: i18n.translate('kbn.advancedSettings.docTableHideTimeColumnTitle', {
+        defaultMessage: 'Hide \'Time\' column',
+      }),
+      value: false,
+      description: i18n.translate('kbn.advancedSettings.docTableHideTimeColumnText', {
+        defaultMessage: 'Hide \'Time\' column in Discover and Saved Searches Dashboard.',
```

Comment:
```suggestion
        defaultMessage: 'Hide the \'Time\' column in Discover and in Saved Searches on Dashboards.',
```

I think this reads a little better.

Reply:
Agreed. Or to be even more explicit: "...in Discover and in all Saved Searches on Dashboards."

Reply:
Good point, updated my suggestion